ducktape/tests/runner_client.py

```diff
@@ -152,6 +153,10 @@ def run(self):
                 if service_errors:
                     summary += "\n\n" + service_errors
 
+            # only check node utilization on test pass
+            if (test_status == PASS):
+                test_status = self._check_cluster_utilization(test_status)
```

Comment:
Please also update the `summary` field with the message.
You can also do it for failed tests as well - just make sure the test summary includes both the original failure and the utilization message